test/run_test.py

```diff
@@ -320,6 +320,12 @@ def skip_test_p(name: str) -> bool:
     'test_fx',  # gets SIGKILL
     'test_dataloader',  # frequently hangs for ROCm
     'test_serialization',   # test_serialization_2gb_file allocates a tensor of 2GB, and could cause OOM
+    'test_utils', # OOM
```

Comment:
If you think there are too many of them here, we could consider run slow gradcheck job serially always.  It would be slower for sure, but, hey, it's periodic :)

cc @clee2000